docs/test/swarm_aux.py

```diff
@@ -3,6 +3,14 @@
 '''
 import underworld as uw
 import numpy as np
+import h5py 
+from underworld.scaling import get_coefficients
+from underworld.scaling import units as u
+from underworld.scaling import non_dimensionalise
+from underworld.scaling import dimensionalise
+
+scaling_coefficients = get_coefficients()
+scaling_coefficients["[length"] = 100 * u.km
```

Comment:
@rbeucher does this do anything?  It doesn't appear to be tested in any case.  I'd suggest pulling it if it isn't tested.

Reply:
No it does not actually, thanks for removing this!